yellowbrick/classifier/classification_report.py

```diff
@@ -106,14 +113,21 @@ def score(self, X, y=None, **kwargs):
 
         # Calculate the percentage for the support metric
         self.support_score = scores[-1]
-        support_percent = self.support_score / (sum(self.support_score))
+        self.support_percent = self.support_score / (sum(self.support_score))
```

Comment:
I do very much like the idea of storing the support scores on the visualizer for access, even if they're not being displayed in the final visualization. However, I don't see any reason why we have to store it in a property that is separate from `scores_`. I propose the following, which might make things a bit simpler. 

```python 
if self.support == PERCENT:
    # scores is a tuple, convert to a list to modify the support score to a percentage
    # note that the support metric is the last item in the scores list 
    scores = list(scores) 
    scores[-1] = scores[-1] / scores[-1].sum()
  
# Create a mapping of class to score for each of the metrics in the scores list 
# Then map the score type (f1, precision, recall, and support) to the dict of class to score. 
scores = map(lambda s: dict(zip(self.classes_, s)), scores) 
self.scores_ = dict(zip(SCORE_KEYS, scores))
```

A copule of points here:

- We should have string constants for `PERCENT` and `COUNT` rather than using `"percent"` and `"count"`. 
- This method uses the np function of sum in `scores[-1].sum()` 
- This version ensures that `self.scores_` contains _all_ score metrics, we'll do the display filtering in draw. 
- Note that this code replaces lines 114-130

Reply:
NOTE: the above is not a complete solution; just a suggestion for keeping all the scores in the same place; we still have to figure out how/where to compute the percent if `not self.support == PERCENT`.

Reply:
@bbengfort 
Still working through this... 😢 
Here's what I understand:
1. The support percentage is needed for all support conditions [True, 'count', 'percent']
2. The only difference between these support conditions [True, 'count', 'percent']  is when you need to draw the heatmap
3.  To draw [True or 'count'], you need a reference to the original support count
4.  To draw 'percent' nothing is needed
My question is how do you store the original support count?

Reply:
I've been thinking about this a bit more, what if we do this:

- `scores_` contains whatever the _display text_ is going to be (either count or percentage) - because this is what users will look at and we want it to be consistent with what they see. 
- Create a property called `_support_percent` (as you did originally) that will hold the percentage for the color display. 

I realize that this is close to what you we're doing before, though exactly opposite to the original method. The alternative though is to have two duplicated data structures, e.g. `scores_` and `scores_intensity_` -- which would be identical except for the case where support is a count. 

What do you think?

```python 
# note that the support metric is the last item in the scores list 
self._support_percent  = scores[-1] / scores[-1].sum()

if self.support == PERCENT:
    # scores is a tuple, convert to a list to modify the support score to a percentage
    scores = list(scores) 
    scores[-1] = self._support_percent
  
# Create a mapping of class to score for each of the metrics in the scores list 
# Then map the score type (f1, precision, recall, and support) to the dict of class to score. 
scores = map(lambda s: dict(zip(self.classes_, s)), scores) 
self.scores_ = dict(zip(SCORE_KEYS, scores))
```

Reply:
We might need to go back and forth on this for a bit.

I thought ```scores_``` contains percentages for the heatmap while the draw method contains the text labels.

Reply:
As it's currently implemented that's true - but this never became an issue until we added `support` -- before that `scores_` was both the label and the heatmap intensity.